CMakeLists.txt

```diff
@@ -63,7 +63,6 @@ if ("${CMAKE_GENERATOR}" MATCHES "Visual Studio")
   else (DEBUG)
     set(CMAKE_CONFIGURATION_TYPES "RelWithDebInfo" CACHE STRING "" FORCE)
   endif (DEBUG)
-  # we want to use the _LOCATION_<config> property
```

Comment:
We still use location_suffix for other vars so I think the better change is to keep the comment but remove LOCATION: sthg like `We want to use the <VARIABLE>_<CONFIG> variants of config-dependent variables.`

Ditto below.

Reply:
Ok